packages/neuron-ui/src/components/Transaction/index.tsx

```diff
@@ -27,6 +35,9 @@ const Transaction = () => {
   const [transaction, setTransaction] = useState(transactionState)
   const [isMainnet, setIsMainnet] = useState(false)
   const [error, setError] = useState({ code: '', message: '' })
+  const [showLockInfoDialog, setShowLockInfoDialog] = useState(false)
```

Comment:
These three states can be reduced to one, `lock | null`, `null` means no lock script is shown, and address can be calculated in UI on the fly